packages/client/src/components/Footer/Footer.tsx

```diff
@@ -29,6 +29,7 @@ const FirstColumn = () => (
         <PhoneNumber
           darkBackground
           eventName={sections.FOOTER}
+          underline
```

Comment:
underline should not be a property of PhoneNumber imho. If you want to render the phone number in print for example, it does not make sense to underline. Underline is a property of a link, not a piece of text.

Reply:
I understand what you mean, I will place the underline inside the Link. The PhoneNumber, however, still needs an underline prop